src/wrapper.rs

```diff
@@ -209,7 +222,22 @@ impl Scalar {
             )),
         }
     }
+    fn __int__(&self) -> BigUint {
+        // Bug, Fr::to_string will print nothing if the value is zero
+        BigUint::from_str(&*self.0.to_string()).unwrap_or(BigUint::ZERO)
```

Comment:
Also not blocking: I've tried to avoid the str method in arkworks in place of using hex since the .to_string method had weird behaviours last time I checked and the way it was doing the conversion seemed non-optimal

Reply:
Good call. Made this PR:

* https://github.com/crate-crypto/py-arkworks-bls12381/pull/8